manager/orchestrator/replicated/update_test.go

```diff
@@ -284,10 +284,6 @@ func testUpdaterRollback(t *testing.T, rollbackFailureAction api.UpdateConfig_Fa
 	assert.Equal(t, observedTask.Status.State, api.TaskStateNew)
 	assert.Equal(t, observedTask.Spec.GetContainer().Image, "image1")
 
-	observedTask = testutils.WatchTaskCreate(t, watchCreate)
```

Comment:
I had an elaborate explanation ready to go for this, but I went to verify it, and it turned out to be incorrect. The updater actually does roll back all three tasks. I've restored the check.